daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -447,6 +447,12 @@ object SExpr {
   final case class LfDefRef(ref: DefinitionRef) extends SDefinitionRef
   // references to definitions generated by the Speedy compiler
   final case class ChoiceDefRef(ref: DefinitionRef, choiceName: ChoiceName) extends SDefinitionRef
+  final case class ChoiceByKeyDefRef(ref: DefinitionRef, choiceName: ChoiceName)
```

Comment:
Does `Profile.scala` need to be extended to handle `ChoiceByKeyDefRef` and friends (as is done for `ChoiceDefRef`) ?